core/src/main/scala/org/apache/spark/sql/execution/command/tables.scala

```diff
@@ -109,6 +115,13 @@ case class TiDescribeTablesCommand(
 
       result
     } else {
+      if (TiAuthorization.enableAuth) {
```

Comment:
I would suggest you call whatever methods directly and check if auth is enabled within TiAuthorization. This way enableAuth is not leaked outside and people using TiAuthorization don't event have to know about it.

Reply:
ACK